src/blt_mad/bltMad.go

```diff
@@ -42,3 +41,17 @@ func BltMad(data []float64, tau float64) []float64 {
 
 	return outliers
 }
+
+func IsAnOutlierBLT(data []float64, tau float64, point float64) bool {
+	//calculate blt formula
+	//everything is based on the noZeroData since we are looking at spikes rather than lack of messages
+	noZeroData := RemoveZeros(data)
+	med := FindMedian(noZeroData)
```

Comment:
What happens if the result of `RemoveZeros` is an empty slice? (asking because the function isn't returning an error and continues execution)